test/lib/jdk/test/lib/process/OutputBuffer.java

```diff
@@ -146,14 +147,18 @@ public String getStderr() {
 
     @Override
     public int getExitValue() {
+      Integer exitCode = this.processExitCode;
```

Comment:
Do we really need the local `exitCode` variable? Even if another multiple threads write to processExitCode, I expect them to all write a non-null Integer.

Reply:
I don't think such an optimization is needed here. This is not performance-critical code.